templates/blog/product-cards/_financialservices-awerness.html

```diff
@@ -0,0 +1,13 @@
+<div class="p-card js-product-card u-hide">
+  <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/dcbd63b9-9outof10.JPG" alt="9outof10financialinstitutions">
```

Comment:
```suggestion
  <img src="https://assets.ubuntu.com/v1/dcbd63b9-9outof10.JPG" alt="9outof10financialinstitutions">
```
remove this class to make the image full sidebar width